sass/app.scss

```diff
@@ -4,3 +4,70 @@
 $fa-font-path: "//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts";
 @import "font-awesome";
 @import "box-sizing";
+
+$Button-Grey: #ddd;
+$Button-Green: #80cbc4;
+$Header-Bar-Blue: #00bbd3;
+$Title-White: #fff;
+$Regular-Text: rgba(255, 255, 255, 0.6);
+$Placeholder-Text: rgba(255, 255, 255, 0.4);
+$Green-Text: #92cbc5;
+$Button-Text-Grey: #1c1c1c;
+$Background-Black: #242524;
+
+.container {
+  width: (480px);
```

Comment:
The container on this will be full width. Resize your browser to make it phone sized.

This example is styled to iOS instead of material, but this is the idea.

![screen shot 2015-11-30 at 5 26 02 am](https://cloud.githubusercontent.com/assets/2532004/11470453/4ccf16c2-9723-11e5-972c-38d76c22925a.png)